.travis.yml

```diff
@@ -160,6 +160,13 @@ matrix:
         - npm run e2e -- --protractor-config=e2e/protractor-ci.conf.js
       env: NAME="Build workbench-web-ng"
 
+
+    # Test submarine-server
+    - language: java
+      jdk: "openjdk8"
+      dist: xenial
+      env: NAME="Test submarine-server" PROFILE="-Phadoop-2.9" BUILD_FLAG="clean package install -DskipTests" TEST_FLAG="test -DskipRat -am" MODULES="-pl ${EXCLUDE_WORKBENCH},${EXCLUDE_INTERPRETER}" TEST_MODULES="-pl submarine-server/server-core" TEST_PROJECTS=""
```

Comment:
There is another test named "Test submarine-server" in line 107 as well. 
It looks like their functionalities are the same. Can we combine them?

Reply:
@yuanzac line 107 is "Test workbench-server". Not is "Test submarine-server".